python/foxglove-sdk-examples/ws-asset-server/main.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import time
+
+import foxglove
+
+
+def asset_handler(uri: str) -> bytes | None:
+    """
+    This will respond to "package://" asset requests from Foxglove by reading files from disk.
+    This example doesn't do any path validation or upward traversal prevention.
+    """
+    logging.debug(f"Asset request: {uri}")
+    if uri.startswith("package://"):
+        filepath = uri.replace("package://", "", 1)
```

Comment:
package:///etc/passwd

I know it's an example, but maybe add a comment that this isn't secure for simplicity as an example

Reply:
>     This example doesn't do any path validation or upward traversal prevention.

That's what I'd added; should I call it out more?

Reply:
Nope, that will do the job